src/bootstrap/doc.rs

```diff
@@ -1027,10 +1027,11 @@ impl Step for RustcBook {
         if self.validate {
             cmd.arg("--validate");
         }
-        if !builder.unstable_features() {
-            // We need to validate nightly features, even on the stable channel.
-            cmd.env("RUSTC_BOOTSTRAP", "1");
-        }
+        // We need to validate nightly features, even on the stable channel.
+        // Set this unconditionally as the stage0 compiler may be being used to
```

Comment:
```suggestion
        // Set this unconditionally as the stage0 compiler may be be used to
```

Reply:
Isn't `may be being used` correct? I was trying to write in the present tense and I think `may be used` would be future tense

Reply:
I'm not a native speaker. I've first made this comment because it sounded wrong to me, but according [to the internet](https://boards.straightdope.com/t/be-being-grammatically-correct/611322/6) it is [correct use](https://forum.wordreference.com/threads/may-be-being-3rd-form-of-verb.2986231/) although people do say that it's not very commonly used.